TwittariaApp/src/main/java/tweets/ITweetCollection.java

```diff
@@ -0,0 +1,11 @@
+package tweets;
+
+import java.util.List;
+import java.util.Map;
+
+public interface ITweetCollection {
+  boolean add(Map<String, Object> tweet);
+  boolean delete(String id);
+  Map<String, Object> get(String id);
```

Comment:
It's better to have POJO instead of Map
POJO - plain old java object, means entity class with getters/setters

Reply:
Ouch, you basically have it bellow, it could be part of service interface.
Ass the convertation and parsing usually are Servlet layer responsibilities